Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -156,6 +156,15 @@ final class UIViewExtensionsTests: XCTestCase {
         XCTAssertEqual(view.shadowOpacity, 0.5)
     }
 
+    func testMasksToBounds() {
+        let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+        let view = UIView(frame: frame)
```

Comment:
No need to create a frame for this test.
```suggestion
        let view = UIView(frame: .zero)
```